index.js

```diff
@@ -1,14 +1,21 @@
 'use strict';
 
-// localCompare works in Node 0.10, but broken in 0.12 and io.js
 var brokenLocaleCompare = 'a'.localeCompare('å') === -1;
+var compare = null;
 
-function compare(a, b) {
-	if (brokenLocaleCompare) {
+if (brokenLocaleCompare) {
+	compare = function (a, b) {
 		return a > b ? 1 : a < b ? -1 : 0;
-	}
-
-	return a === b ? 0 : a.localeCompare(b);
+	};
+} else if (typeof Intl.Collator === 'function') {
+	var collator = new Intl.Collator({sensitivity: 'variant'});
```

Comment:
Following the documentation, the ```variant``` has the largest range of differences between strings. It's similar to the ```String.prototype.localeCompare```. 
Do you disagree?